0002121980ABBAC9.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "0002121980ABBAC9",
+    "servers": [
+      {
+        "server_address": "asia-se.thethings.network",
```

Comment:
I believe this server address is incorrect. Check the list here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses

Reply:
Thank for your help, i did change server address and send it back to you, please make a review!
Best Regards!